
Taken out of X.org CVS to fix nas relocation errors on e.g. powerpc and
x86-64.

  - Rene Rebe <rene@exactcode.de>

===================================================================
RCS file: /cvs/xorg/xc/lib/Xau/Imakefile,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- xc/lib/Xau/Imakefile	2004/04/23 18:43:36	1.2
+++ xc/lib/Xau/Imakefile	2004/10/12 19:13:42	1.3
@@ -37,6 +37,9 @@
 OBJS =	AuDispose.o AuFileName.o AuGetAddr.o AuGetBest.o AuLock.o \
 	AuRead.o AuUnlock.o AuWrite.o $(K5ENCOBJ)
 
+#define IncludeSharedObjectInNormalLib
+#define UnsharedLibObjCompile(options) LibObjCompile(unshared,$(CDEBUGFLAGS) $(CLIBDEBUGFLAGS) options $(PICFLAGS))
+
 #include <Library.tmpl>
 
 DependTarget()
